GVRf/Framework/jni/shaders/material/mesh_animation_shader.cpp

```diff
@@ -0,0 +1,133 @@
+#include "mesh_animation_shader.h"
```

Comment:
Ok. I will clean up the previous implementation. Thanks @fredduarte for contributing it again. It really helps The crash is fixed -- I added one line to handle a corner case explicitly.